Source/Core/VideoBackends/D3D12/Globals.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2008 Dolphin Emulator Project

```

Comment:
I doubt there is a use for such an including header file, at least it's not often used.